go.sum

```diff
@@ -211,6 +211,8 @@ github.com/vmihailenco/tagparser/v2 v2.0.0 h1:y09buUbR+b5aycVFQs/g70pqKVZNBmxwAh
 github.com/vmihailenco/tagparser/v2 v2.0.0/go.mod h1:Wri+At7QHww0WTrCBeu4J6bNtoV6mEfg5OIWRZA9qds=
 github.com/vmware/vcf-sdk-go v0.4.1 h1:Hd5flatP0xg0oEBcEabS6zYcMm6doVePJXzj4gfrfus=
```

Comment:
I wonder why these are not removed?

Reply:
It needed a `go mod tidy` ran. I've pushed an update for @spacegospod.